src/textual/renderables/bar.py

```diff
@@ -5,7 +5,7 @@
 from rich.text import Text
 
 
-class UnderlineBar:
+class RenderableBar:
```

Comment:
I'd rather not have "Renderable" in the name, since they are all renderables. I guess you renamed it, because you repurposed it as a progress bar. How about just "Bar" then?